src/api/app/views/webui2/webui/package/_rpmlint_log.html.haml

```diff
@@ -0,0 +1,7 @@
+- @log.lines.each do |line|
+  - if (line =~ /\w+(?:\.\w+)+: W: /)
```

Comment:
Using `match?` is more performant in this case. See https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Performance/RegexpMatch